src/greenlight/runner.clj

```diff
@@ -112,27 +112,22 @@
   [^ExecutorService exec-pool system tests]
   {:post [(instance? Reader (first %))
           (instance? Future (second %))]}
-  (let [reader (PipedReader.)
-        writer (PipedWriter. reader)
-        bindings (get-thread-bindings)
-        group-f (reify Callable
-                  (call [_]
-                    (with-bindings bindings
-                      (binding [*out* writer, *err* writer]
-                        (try
-                          (mapv (partial test/run-test! system) tests)
-                          (finally
-                            (.close writer)))))))]
+  (let [writer (PipedWriter.)
+        reader (PipedReader. writer (int (Math/pow 2 20)))
+        group-f (bound-fn
```

Comment:
Does this need to be `bound-fn` as opposed to `fn`? I don't think you're looking up any vars that were bound outside of scope.

Reply:
`test/*report*` is thread bound, yeah.

Reply:
Ah, carry on